aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +258,40 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Full-page form
+
+- Use full-page forms for create and edit flows, irrespective of the size of the form.
+- Use a single-column full-page form instead of multi-column layouts.
+- Full-page forms shoud be center-aligned.
+- Use full-page when the form is too long.
```

Comment:
"too long" seems arbitrary. What is too long?

Reply:
Some research suggested  12 and some 10 so i just put too long lol. 😔 Option "Display no more than 10 form fields with in the same viewport." Or "Minimize the number of form fields so users see within the same viewport." Suggestions ...